backends-common/rabbitmq/src/main/java/org/apache/james/backend/rabbitmq/RabbitMQConfiguration.java

```diff
@@ -102,13 +107,16 @@ public final int hashCode() {
         private final URI amqpUri;
         private final URI managementUri;
         private final ManagementCredentials managementCredentials;
+        private final CassandraMailQueueViewConfiguration cassandraMailQueueViewConfiguration;
         private Optional<Integer> maxRetries;
         private Optional<Integer> minDelay;
 
-        private Builder(URI amqpUri, URI managementUri, ManagementCredentials managementCredentials) {
+        private Builder(URI amqpUri, URI managementUri, ManagementCredentials managementCredentials,
+                        CassandraMailQueueViewConfiguration cassandraMailQueueViewConfiguration) {
             this.amqpUri = amqpUri;
             this.managementUri = managementUri;
             this.managementCredentials = managementCredentials;
+            this.cassandraMailQueueViewConfiguration = cassandraMailQueueViewConfiguration;
```

Comment:
No! We should not include a specific conf in a generic one...

Please note that sharing the same file != handling the conf reading in the same place.